bamazonCustomer.js

```diff
@@ -0,0 +1,132 @@
+var mysql = require("mysql");
+var inquirerUserPrompt = require("inquirer");
+var Table = require('cli-table');
+
+var connection = mysql.createConnection({
+  host: "localhost",
+  port: 3306,
+
+  // Your username
+  user: "root",
+
+  // Your password
+  password: "",
+  database: "bamazon_db"
+});
+
+connection.connect(function(err) {
+  if (err) {
+    throw err;
+  };
+  console.log("");
+});
+
+function showAll() {
+  var table = new Table({
+      head: ['Item ID', 'Product Name', 'Department Name', 'Price', 'Stock Quantity']
+    , colWidths: [11, 30, 30, 14, 16]
+  });
+  connection.query("SELECT * FROM products, department where products.department_id = department.department_id", function(err, res) {
+    if (err) {
+      throw err;
+    };
+    for (var i = 0; i < res.length; i++) {
+      table.push([res[i].item_id, res[i].product_name, res[i].department_name, res[i].price, res[i].stock_quantity]);
+
+    }
```

Comment:
this could be rewritten to be a little easier to read like this:
```javascript
res.map(function(item){
  table.push([
    item.item_id, item.product_name, item.department_name, item.price, item.stock_quantity
  ])
})